src/rebloom.c

```diff
@@ -517,9 +517,9 @@ static int cfInsertCommon(RedisModuleCtx *ctx, RedisModuleString *keystr, RedisM
             } else {
                 RedisModule_ReplyWithLongLong(ctx, -1);
             }
-        } else {
+        } else if (insStatus == CuckooInsert_MemAllocFailed){                   // LCOV_EXCL_LINE
```

Comment:
This series of if-statements over insStatus really belongs in a switch.

Reply:
@rafie As requested